pkg/util/injection.go

```diff
@@ -39,24 +39,24 @@ func FixInjectionsWithRelativePath(workdir string, injections *api.InjectionList
 
 // ExpandInjectedFiles returns a flat list of all files that are injected into a
 // container. All files from nested directories are returned in the list.
-func ExpandInjectedFiles(injections api.InjectionList) ([]string, error) {
+func ExpandInjectedFiles(injections api.VolumeList) ([]string, error) {
 	result := []string{}
 	for _, s := range injections {
-		info, err := os.Stat(s.SourcePath)
+		info, err := os.Stat(s.Source)
 		if err != nil {
 			return nil, err
 		}
 		if !info.IsDir() {
-			return nil, fmt.Errorf("the %q must be a valid directory", s.SourcePath)
+			return nil, fmt.Errorf("the %q must be a valid directory", s.Source)
```

Comment:
I thought the destination had to be a dir, why does the source have to be a dir?